pkg/machines/index.es6

```diff
@@ -23,7 +23,7 @@ import App from './app.jsx';
 import { getAllVms } from './actions.es6';
 import { logDebug } from './helpers.es6';
 
-function render() {
+function render () {
```

Comment:
Unnecessary change. In particular in our code style we usually have functions without a space after their name.